x/execution/client/rest/query.go

```diff
@@ -62,7 +63,57 @@ func queryListHandlerFn(cliCtx context.CLIContext) http.HandlerFunc {
 
 		route := fmt.Sprintf("custom/%s/%s", types.QuerierRoute, types.QueryList)
 
-		res, height, err := cliCtx.QueryWithData(route, nil)
+		filter := types.ListFilter{}
+		if param := r.FormValue("parentHash"); param != "" {
+			h, err := hash.Decode(param)
+			if err != nil {
+				rest.WriteErrorResponse(w, http.StatusBadRequest, err.Error())
+				return
+			}
+			filter.ParentHash = h
+		}
+
+		if param := r.FormValue("eventHash"); param != "" {
+			h, err := hash.Decode(param)
+			if err != nil {
+				rest.WriteErrorResponse(w, http.StatusBadRequest, err.Error())
+				return
+			}
+			filter.EventHash = h
+		}
+
+		if param := r.FormValue("instanceHash"); param != "" {
+			h, err := hash.Decode(param)
+			if err != nil {
+				rest.WriteErrorResponse(w, http.StatusBadRequest, err.Error())
+				return
+			}
+			filter.InstanceHash = h
+		}
+
+		if param := r.FormValue("processHash"); param != "" {
+			h, err := hash.Decode(param)
+			if err != nil {
+				rest.WriteErrorResponse(w, http.StatusBadRequest, err.Error())
+				return
+			}
+			filter.ProcessHash = h
+		}
+
+		if param := r.FormValue("status"); param != "" {
```

Comment:
is `status` a number or a string in the request?

Reply:
This is a string, I updated the description